base/workerpool.jl

```diff
@@ -3,6 +3,9 @@
 
 type WorkerPool
     channel::RemoteChannel{Channel{Int}}
+
+    # Create a shared queue of available workers...
+    WorkerPool() = new(RemoteChannel(()->Channel{Int}(128)))
```

Comment:
This should be changed to `RemoteChannel(()->Channel(typemax(Int)))`, else ` WorkerPool(workers::Vector{Int})` will block when trying to add more than 128 workers.